flink-core/src/main/java/org/apache/flink/configuration/ConfigOption.java

```diff
@@ -73,11 +73,11 @@
 
 	// ------------------------------------------------------------------------
 
-	Class getClazz() {
+	public Class getClazz() {
 		return clazz;
 	}
 
-	boolean isList() {
+	public boolean isList() {
```

Comment:
Unless we believe that these methods provide value to users I'm actually inclined to call these via reflection.